File: comms/push-notifications/credentials/delete-credential/delete-credential.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "<username>",
        authToken: "<password>",
    });
    await client.pushNotifications.credentials.delete("comms_credential_01h9krwprkeee8fzqspvwy6nq8");
}
main();

File: comms/push-notifications/credentials/delete-credential/delete-credential.rb
require "twilio"

client = Twilio::Client.new(
  account_id: "<username>",
  auth_token: "<password>"
)

client.push_notifications.credentials.delete(credential_id: "comms_credential_01h9krwprkeee8fzqspvwy6nq8")
